File: SDKs/nexmo.com/messages-olympus/1.3.2/typescript/src/sdk/models/shared/errorsender.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// ErrorSender
/** 
 * Invalid sender
**/
export class ErrorSender extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=detail" })
  detail: string;

  @SpeakeasyMetadata({ data: "json, name=instance" })
  instance: string;

  @SpeakeasyMetadata({ data: "json, name=title" })
  title: string;

  @SpeakeasyMetadata({ data: "json, name=type" })
  type: string;
}

File: SDKs/nexmo.com/messages-olympus/1.3.2/java/lib/src/main/java/openapisdk/models/shared/ErrorSender.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * ErrorSender
 * Invalid sender
**/
public class ErrorSender {
    @JsonProperty("detail")
    public String detail;
    public ErrorSender withDetail(String detail) {
        this.detail = detail;
        return this;
    }
    @JsonProperty("instance")
    public String instance;
    public ErrorSender withInstance(String instance) {
        this.instance = instance;
        return this;
    }
    @JsonProperty("title")
    public String title;
    public ErrorSender withTitle(String title) {
        this.title = title;
        return this;
    }
    @JsonProperty("type")
    public String type;
    public ErrorSender withType(String type) {
        this.type = type;
        return this;
    }
}